client/src/components/App/Content/InvestigationForm/TabManagement/ExposuresAndFlights/ExposureForm/ExposureForm.tsx

```diff
@@ -1,47 +1,71 @@
+import { format } from "date-fns";
+import { Grid } from "@material-ui/core";
+import React from "react";
 
-const ExposureForm = () => {
-    const placeholderText = 'הכנס שם...';
-    const classes = useFormStyles();
+import useFormStyles from "styles/formStyles";
+import DatePick from "commons/DatePick/DatePick";
+import CircleSelect from "commons/CircleSelect/CircleSelect";
+import { dateFormatForDatePicker } from "Utils/displayUtils";
+import CircleTextField from "commons/CircleTextField/CircleTextField";
+import FormRowWithInput from "commons/FormRowWithInput/FormRowWithInput";
+import LocationInput, {GoogleApiPlace} from 'commons/LocationInputField/LocationInput';
+import PlacesTypesAndSubTypes from "commons/Forms/PlacesTypesAndSubTypes/PlacesTypesAndSubTypes";
 
-    const { exposureData, setExposureData } = useContext(exposuresContext);
-    const { exposingPersonName, placeType, exposureLocation } = exposureData;
-    const { exposingPersonName: setExposingPersonName, placeType: setPlaceType, exposureLocation: setExposureLocation } = setExposureData;
+const ExposureForm = (props: any) => {
+  const {
+    exposureAndFlightsData,
+    fieldsNames,
+    handleChangeExposureDataAndFlightsField,
+  } = props;
 
-    const placeTypeOptions = [
-        { id: 1, name: 'מקום ציבורי' },
-        { id: 2, name: 'מקום דת' },
-        { id: 3, name: 'מקום מקומי' }
-    ];
+  const classes = useFormStyles();
 
-    const selectPlaceType = (event: React.ChangeEvent<any>) => setPlaceType(event.target.value);
-    const handlePersonNameInput = (event: React.ChangeEvent<HTMLInputElement>) => setExposingPersonName(event.target.value);
-    const onLocationChange = (event: React.ChangeEvent<{}>, newValue: GoogleApiPlace | null) => setExposureLocation(newValue);
+  return (
+    <Grid className={classes.form} container justify="flex-start">
+      <FormRowWithInput fieldName="שם החולה:">
+        <>
+          <CircleTextField
+            value={exposureAndFlightsData[fieldsNames.firstName]}
+            onChange={(e) =>
+              handleChangeExposureDataAndFlightsField(fieldsNames.firstName, e.target.value)
+            }
+            placeholder="שם פרטי..."
+          />
+          <CircleTextField
+            value={exposureAndFlightsData[fieldsNames.lastName]}
+            onChange={(e) =>
+              handleChangeExposureDataAndFlightsField(fieldsNames.lastName, e.target.value)
+            }
+            placeholder="שם משפחה..."
+          />
+        </>
+      </FormRowWithInput>
 
-    return (
-        <Grid className={classes.form} container justify='flex-start'>
-            <FormRowWithInput fieldName='שם החולה:'>
-                <CircleTextField value={exposingPersonName} onChange={handlePersonNameInput}
-                    placeholder={placeholderText} test-id='confirmedPatientName' />
-            </FormRowWithInput>
+      <FormRowWithInput fieldName="תאריך החשיפה:">
+        <DatePick
```

Comment:
![image](https://user-images.githubusercontent.com/24894463/93024495-b6c43900-f5ff-11ea-8e68-4434af5d8f81.png)

days are not displayed on my screen